### PR TITLE
Fix for item quantity value in transactional emails when partially or fully canceled

### PR DESCRIPTION
**Pull request by kl3m3n**
### Description (*)
When you partially or fully cancel items from order then the wrong item quantity is shown in transactionals emails. The price is correct but the quantity is not. This fix adjust item quantity to correct amount in all transactional emails that uses layout handle "sales_email_order_items".

### Manual testing scenarios (*)
1. Create order with multiple items with 1 or more quantity
2. Cancel some items fully or cancel some items partially
3. Send order email from admin (or programatically) and you'll get original item quantity despite being canceled.

### Questions or comments
Affected line is 35 changed from 
`<td class="item-qty"><?= /* @escapeNotVerified */  $_item->getQtyOrdered() * 1) ?></td>`
to
`<td class="item-qty"><?= /* @escapeNotVerified */  ($_item->getQtyOrdered() * 1) - ($_item->getQtyCanceled() * 1) ?></td>`

### Contribution checklist (*)
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=23155) <br/>All committers have signed the CLA.

**Comment by davidverholen** [comment]
Hi @kl3m3n ,

thank you for your contribution. In this repository the Magento code resides in the app/code directory and so the change has to be made there (That's why there is a whole new file instead of just a diff of the one line in the changes)

So for us to be able to process this pr, please make the change here:
https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Sales/view/frontend/templates/email/items/order/default.phtml

If you need help you can also have a look at the relatively new contribution beginners guide: https://github.com/magento/magento2/wiki/Getting-Started

**Comment by orlangur** [comment]
@davidverholen why we need `on hold` here?

**Comment by davidverholen** [comment]
@orlangur just wanted to make sure its not processed further until the change is in the correct path

**Comment by orlangur** [comment]
@davidverholen it does not work this way yet, single approval will remove both labels :(

**Comment by davidverholen** [comment]
@orlangur yes, I know :/ My hope was, that people are more careful with approving when the on hold label is set

**Review by davidverholen — CHANGES_REQUESTED** [review]
move changes to correct file location at https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Sales/view/frontend/templates/email/items/order/default.phtml

**Comment by davidverholen** [comment]
@orlangur but right, I can request changes in a review which will block it also when someone else approves it. I will hide our comments now to reduce noise here

**Pull request closed without merging**
